spec/abci/abci++_app_requirements.md

```diff
@@ -810,9 +810,21 @@ implementation of
 
 On startup, CometBFT calls the `Info` method on the Info Connection to get the latest
 committed state of the app. The app MUST return information consistent with the
-last block it successfully completed Commit for.
+last block it successfully completed `Commit` for. 
 
-If the app successfully committed block H, then `last_block_height = H` and `last_block_app_hash = <hash returned by Commit for block H>`. If the app
+The application is expected to persist its state during the `Commit` method.
+CometBFT and the application are expected to crash together and there should not 
+exist a scenario where the application has persisted state that CometBFT has not.
+
+In other words, if `Commit` for a block H has not been executed and CometBFT crashed,
+the application
+should not have persisted any state between the `Commit` for H - 1 and 
```

Comment:
It's actually H-N (I know it's surprising, but it's like that). The App does not need to persist every height